src/js/components/tasks/cards/card.tsx

```diff
@@ -125,10 +125,10 @@ const OrgCard = ({
   );
   const doRefreshOrg = useCallback(() => {
     /* istanbul ignore else */
-    if (org?.org_type === ORG_TYPES.QA && handleRefresh) {
+    if (org && org.org_type === ORG_TYPES.QA && handleRefresh) {
```

Comment:
Aw, no `?.`.

Reply:
We're still using it most places... but there are some bugs in eslint-plugin-react hooks (https://github.com/facebook/react/issues/18926, https://github.com/facebook/react/issues/19043) which cause this to break at the moment.